package.json

```diff
@@ -1,8 +1,15 @@
 {
-  "name": "@postman/schemas",
-  "version": "0.0.1",
+  "name": "postman-schemas",
+  "version": "1.0.0",
```

Comment:
Let the version be as is, this does not represent the collection schemas themselves. Also, versions should not be bumped in pull requests.